app/src/main/res/layout/item_repo.xml

```diff
@@ -1,67 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-              android:layout_height="wrap_content"
-              android:orientation="vertical">
+    android:layout_height="wrap_content"
+    android:orientation="vertical">
 
     <android.support.v7.widget.CardView
         android:id="@+id/card_view"
         style="@style/CardViewContainer"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        >
+        android:layout_height="wrap_content">
 
         <LinearLayout
             android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:orientation="vertical">
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:paddingBottom="4dp"
+            android:paddingLeft="8dp"
+            android:paddingRight="8dp"
+            android:paddingTop="4dp">
 
             <TextView
                 android:id="@+id/item_repo_name"
-                style="@style/ViewBuildText.TwoLines.Title.NoImage"
+                style="@style/ViewBuildText.TwoLines.Big"
                 android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                />
+                android:layout_height="wrap_content" />
 
-            <LinearLayout
+            <TextView
+                android:id="@+id/item_repo_duration"
+                style="@style/ViewBuildText"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:orientation="horizontal">
-
-                <ImageView
-                    style="@style/ViewBuildImageIcon"
-                    app:srcCompat="@drawable/ic_duration_grey600_16dp"
-                    />
+                android:drawableLeft="@drawable/ic_duration_grey600_16dp"
```

Comment:
Maybe I'm not right, but I don't like using drawable params for TextView. I had before some issues with them, when I need to add specific alignment or padding/margin. That's why I had separate ImageView, as you could see before.
I prefer to keep using TextView and ImageView for such cases, even if it looks like only TextView with drawableLeft/drawableRight could be used.